Assets/Scripts/World/SkyboxController.cs

```diff
@@ -8,15 +8,33 @@ public class SkyboxController : MonoBehaviour
     public Light moon;
 
     [SerializeField, Range(0, 24)]
-    private float timeOfDay = 0.0f;
+    private float timeOfDay = 9.0f;
 
     [SerializeField, Range(0.01f, 1.0f)]
     private float timeScale = 1.0f;
 
     public AnimationCurve sunCurve;
     public AnimationCurve moonCurve;
 
+    public bool dayNightCycleEnabled = true;
+
+    private void Start()
+    {
+        // update once to set initial values
+        UpdateSunAndMoon();
+    }
+
     private void Update()
+    {
+        if (!dayNightCycleEnabled)
+        {
+            return;
+        }
+
+        UpdateSunAndMoon();
+    }
+
+    private void UpdateSunAndMoon()
     {
         timeOfDay += Time.deltaTime * timeScale;
         timeOfDay %= 24;
```

Comment:
Since timeOfDay is in expressed in hours, and deltaTime is in seconds, you should do : ´timeOfDay += Time.deltaTime * timeScale / 3600.0;´


This way when ´timeScale = 1´ timeOfDay is moves at the same speed as the time in real life.